ra2ce/graph/network_config_data/network_config_data_validator.py

```diff
@@ -35,6 +35,25 @@
 NetworkDictValues: dict[str, list[Any]] = {
     "polygon": ["file", None],
     "directed": [True, False, None],
+    "network_type": ["walk", "bike", "drive", "drive_service", "all", None],
```

Comment:
This is not required, the validator is checking the `NetworkTypeEnum` inside the `_validate_network_section`, same applies for `road_types` , if you are missing values, then extend the corresponding enums.